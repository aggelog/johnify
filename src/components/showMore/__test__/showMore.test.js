import React from 'react';
import ShowMore from '../index';
import renderer from 'react-test-renderer';

it('renders correctly', () => {
  const tree = renderer
    .create(<ShowMore showMore={jest.fn()} />)
    .toJSON();
  expect(tree).toMatchSnapshot();
});