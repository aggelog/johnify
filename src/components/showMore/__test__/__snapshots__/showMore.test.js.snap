// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders correctly 1`] = `
<div
  className="show-more__row row"
>
  <div
    className="show-more__button show-more__light-shadow"
    onClick={[Function]}
  >
    Give me more
  </div>
</div>
`;
